app/helpers/writable_helper.rb

```diff
@@ -49,10 +49,11 @@ def privacy_state(privacy)
     privacy_icon(privacy) + ' ' + PRIVACY_MAP[privacy][0]
   end
 
-  def privacy_icon(privacy)
+  def privacy_icon(privacy, alt=false)
```

Comment:
Where do we use `privacy_icon`? I have the feeling that we should show alt text by default, and only remove it if we explicitly don't need it (e.g. because we have the text nearby, like in the above `privacy_state` method)

Reply:
I can flip the default, sure.